### ISSUE TITLE
repo2docker-entrypoint: line 8: exec: jupyterhub-singleuser: not found

### ISSUE THREAD

**Issue by bgroenks96** [pangeo-data/pangeo-stacks#132]
I am encountering this error when JupyterHub attempts to spawn the JupyterLab pod since upgrading to the latest version of base-notebook-onbuild 2020.02.07-1c213fa.

Has anyone seen this before? It could be an issue with my setup, but I'm not sure where to look.

**Comment by bgroenks96** [comment]
Seems to be caused by [this file](https://github.com/jupyter/repo2docker/blob/9099def40a331df04ba3ed862ee27a8e4a77fe43/repo2docker/buildpacks/repo2docker-entrypoint) in repo2docker receiving jupyterhub-singleuser as an argument. But I'm still not sure how or why this happens (or what changed; I upgraded the JupyterHub and Pangeo versions).

**Comment by scottyhq** [comment]
@bgroenks96 sounds like you are running this in your own Hub? What helm versions of jupyterhub and pangeo are you running? 

I haven't seen that error before but we have made some version changes to the stacks images recently. I tried the specific image you pointed to above on our binder deployments and am not seeing any errors:
https://github.com/scottyhq/pangeo-binder-test

**Comment by bgroenks96** [comment]
Yes, I have my own fork of pangeohub. I am using JuypterHub 0.9-beta3 (also failed with 0.9-beta2, pangeohub's current version) and the latest version of pangeohub from pangeo-helm repository, with some minor modifications (like the nvidia driver installer). I also have dask-gateway removed because it seems to cause a separate, unrelated error. I'll double check, but everything else should  be exactly the same as in pangeo-helm master.

**Comment by bgroenks96** [comment]
Also, I should mention that this error was resolved for me by downgrading to pangeo 2020.02.06-e705b78 (merge commit from PR #118). The start script functionality didn't seem to work, though (script didn't run), so I ended up just reverting to a version from October.

**Comment by rabernat** [comment]
I am trying an failing to launch a binder with `pangeo/pangeo-notebook-onbuild:2020.02.03-0853dbb` (see https://github.com/cgentemann/osm2020tutorial). The binder won't launch. Could it be related?

**Comment by rabernat** [comment]
nm, that image is not the most recent one
